### PR TITLE
Update sql_database docs for 3.11

### PR DESCRIPTION
**Pull request by djgalvan**
Only tested 3.11, not sure of other versions. `__init__.py` caused issues with imports during testing so I dropped it. Think I covered all the bases, will need someone to review, especially in surrounding instructions.

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 9ccceb22fbfe08067379e9ccea4c094e6db7ddcd at: https://4217b44d.fastapitiangolo.pages.dev

**Comment by djgalvan** [comment]
Relative doc preview at: [https://4217b44d.fastapitiangolo.pages.dev/tutorial/sql-databases/](url)

**Comment by tiangolo** [comment]
📝 Docs preview for commit 0cd61c38f954dc229990b573a825194228ce2fb6 at: https://2edf1d5e.fastapitiangolo.pages.dev

**Review by dtleal — APPROVED** [review]

File: docs_src/sql_databases/sql_app_py311/tests/test_sql_app.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    finally:
+        db.close()
+
+
+app.dependency_overrides[get_db] = override_get_db
+
+client = TestClient(app)
+
+
+def test_create_user():
```
Comment (siavashyj) [inline docs_src/sql_databases/sql_app_py311/tests/test_sql_app.py:31]: can you please add unit tests for the other read user and item crud endpoints?

**Comment by tiangolo** [comment]
📝 Docs preview for commit cf455ce2d1f0d425b478a9ead0d1d55213829124 at: https://269dff4d.fastapitiangolo.pages.dev

File: docs_src/sql_databases/sql_app_py311/tests/test_sql_app.py (hunk 1 above)
Comment (djgalvan) [reply docs_src/sql_databases/sql_app_py311/tests/test_sql_app.py:31]: I couldn't reliably reproduce those tests within the `docs_src/` dir, they exist within `tests/`.

Comment (djgalvan) [reply docs_src/sql_databases/sql_app_py311/tests/test_sql_app.py:31]: looking at all the other directories within `docs_src/`, they don't seem to belong there.

Comment (djgalvan) [reply docs_src/sql_databases/sql_app_py311/tests/test_sql_app.py:31]: I assumed they were probably meant/planned for another doc explaining how to create tests

Comment (djgalvan) [reply docs_src/sql_databases/sql_app_py311/tests/test_sql_app.py:31]: Found where the tests were being documented, `docs/advanced/testing-database.md` I'll have a look at it this week.

**Pull request closed without merging**